libdnf/repo/Repo.hpp

```diff
@@ -173,6 +173,8 @@ struct Repo {
     void setUseIncludes(bool enabled);
     bool getLoadMetadataOther() const;
     void setLoadMetadataOther(bool value);
+    bool getLoadFilelists() const;
+    void setLoadFilelists(bool value);
```

Comment:
May I ask you for a doc string for the new methods? I know that other method does not have it, but it is not correct.